### PR TITLE
small typos in 3.3

### PR DESCRIPTION
**Pull request by sbatson5**
## What's in this PR?

Just spotted some small typos while reading through some of the 3.3.0 stuff.

### CODE REVIEW

File: guides/v3.3.0/configuring-ember/debugging.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ RSVP.on('error', function(error) {
 #### Errors within `Ember.run.later` ([Backburner.js](https://github.com/ebryn/backburner.js))
 
 Backburner has support for stitching the stacktraces together so that you can
-track down where an erroring `Ember.run.later` is being initiated from. Unfortunately,
+track down where an error thrown by `Ember.run.later` is being initiated from. Unfortunately,
```
Comment (sbatson5) [inline guides/v3.3.0/configuring-ember/debugging.md]: Happy to change this one if people disagree but I don't think `erroring` is technically a word.

File: guides/v3.3.0/contributing/adding-new-features.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ module.exports = function(environment) {
 
 ### `features.json`
 
-The root of the repository will contain a features.json file, which will
+The root of the repository will contain a `features.json` file, which will
```
Comment (sbatson5) [inline guides/v3.3.0/contributing/adding-new-features.md]: Some spots in this file wrapped the file name in tick marks and some didn't.

File: guides/v3.3.0/tutorial/service.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ as a starting point when you need to add one.
 
 
 The [Google Maps API](https://developers.google.com/maps/documentation/javascript/tutorial/) requires us to reference its library from a remote script.
-In this case we'll provide this script to our Ember app via an Addon called `ember-simple-google-maps`.
+In this case we'll provide this script to our Ember app via an addon called `ember-simple-google-maps`.
```
Comment (sbatson5) [inline guides/v3.3.0/tutorial/service.md]: `addon` didn't seem to be capitalized anywhere else.

**Comment by jenweber** [comment]
Thank you! These are great.

**Pull request merged**